Include <cmath> for isnan(). Due to the definition of isnan for WIN32 and
__APPLE__, one must alias isnan in the root namespace to
std::isnan(). Hackish: there is probably a better solution.

--- include/jrl/mal/matrixabstractlayercheck.hh~	2011-04-14 22:17:56.000000000 +0200
+++ include/jrl/mal/matrixabstractlayercheck.hh	2011-10-29 13:46:48.000000000 +0200
@@ -34,6 +34,10 @@
 #  endif /*isnan*/
 # elif defined __APPLE__
    inline bool isnan(double x) { return x != x; }
+# else
+#  include <cmath>
+#  undef isnan
+#  define isnan std::isnan
 # endif /*WIN32*/
 
 inline bool malIsNanVector(const vectorN& inVector)
